Bloom.md

```diff
@@ -68,8 +68,9 @@ We have the following terminologies / properties:
 Upon loading, the module registers a new bloom filter module based data type, creates bloom filter (BF.*) commands,
 bloom specific configurations and the bloom ACL category.
 
-* Module name: bloom
+* Module name: bf
```

Comment:
Can you describe why it's good to use the same module name `bf` as ReBloom? Compatibility for clients that check the HELLO, INFO, etc.

Reply:
Yes, I updated the section with the explanation